Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: concourse
 type: application
-version: 17.1.0
+version: 17.2.0
```

Comment:
thx for the PR. You dont need to bump the version. It will be done when publishing.